lib/winston/transports/file.js

```diff
@@ -105,6 +108,10 @@ module.exports = class File extends TransportStream {
       return true;
     }
 
+    if (this.lazy) {
+      this.open();
```

Comment:
I worry whether this affects performance because this gets called on every `log()` call -- we should check and see

Reply:
Right! The first thing `open` does when it's called, it's checking if it was already opened, but it's an overhead of extra function call in every `log` operation. So, I will try to change this to flag check here and measure performance.